src/components/Card.js

```diff
@@ -1,18 +1,28 @@
-import React from "react";
+import React, { useState } from "react";
 import PropTypes from "prop-types";
 import emoji from "emoji-dictionary";
 
 import "./Card.css";
+import axios from "axios";
 //import Axios from "axios";
 
 const Card = (props) => {
+
```

Comment:
This ONE function here is used to pass into the board.js and then the board.js's function called onDeleteClick gets passed here. Basically, line 15 is just the prop being passed.